staticjinja/dep_graph.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding:utf-8 -*-
+
+"""
+Dependency graph for staticjinja
+"""
+
+
+class DepGraph(object):
+    """
+    A directed graph which will handle dependencies between sources in a site.
```

Comment:
This documentation should be more generic-- the class shouldn't be have any knowledge of staticjinja.